src/Headers.ts

```diff
@@ -25,7 +25,7 @@ export class Headers {
      * because that class is only defined in the browser.
      */
     if (
-      ['Headers', 'HeadersPolyfill'].includes(init?.constructor.name) ||
+      ['Headers', 'HeadersPolyfill', '_Headers'].includes(init?.constructor.name) ||
```

Comment:
We should also add a test for this, implicitly. Bumping the used Node.js version in the CI would be enough. I recommend adding a matrix of Node.js versions so we test both older versions and the recent ones that introduce this change. 

Let me know if you need help with this.

Reply:
@kettanaito Done, I think. I have no real experience working with GitHub Actions, so I don't know if my approach is will work and is the best way of doing it.

Reply:
You did it right 👌